File: app/src/main/java/com/casebeaumonde/activities/ClosetItem/response/AddToFavClosetItemResponse.java
package com.casebeaumonde.activities.ClosetItem.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class AddToFavClosetItemResponse {

@SerializedName("code")
@Expose
private String code;

public String getCode() {
return code;
}

public void setCode(String code) {
this.code = code;
}
}

File: app/src/main/java/com/casebeaumonde/activities/ClosetItem/response/AddToFavClosetItemResponse.kt
package com.casebeaumonde.activities.ClosetItem.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class AddToFavClosetItemResponse {
    @SerializedName("code")
    @Expose
    private var code: String? = null

    fun getCode(): String? {
        return code
    }

    fun setCode(code: String?) {
        this.code = code
    }
}